src/js/components/molecules/flyout/Flyout.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+    <ul class="main-nav__flyout"
+        v-if="items && items.length > 0"
+        @keydown.down="selectNextLink"
+        @keydown.tab.exact="selectNextLink"
+        @keydown.up="selectPrevLink"
+        @keydown.tab.shift.exact="selectPrevLink"
+        @keydown.right="selectNextSection($event, true)"
+        @keydown.left="selectPrevSection($event, true)"
+        @mouseenter.stop="open"
+        v-show="isCompact || !viewportUtil.isResizing">
+        <flyout-section ref="items"
+                        v-for="(item, key) in items"
+                        v-if="item.items && item.items.length"
+                        v-bind="item"
+                        :is-compact="isCompact"
+                        :key="key"
+                        :itemId="`flyout_title_${ uniqId }_${ key }`"
+                        @content-link-click="$emit('content-link-click')"
+        >
+        </flyout-section>
+    </ul>
+</template>
+
+<script lang="ts">
+    import Vue from 'vue';
+    import ViewportUtility from '../../../utils/viewport';
+
+    export default Vue.component('Flyout', {
+        props: {
+            items: {
+                type: Array,
+                required: true,
+            },
+            isCompact: {
+                type: Boolean,
+                required: false,
+            },
+            disableKeyboardEvents: {
+                type: Boolean,
+                required: false,
+                default: false,
+            },
+        },
+        data() {
+            return {
+                sectionIndex: 0,
+                viewportUtil: new ViewportUtility(),
+            }
+        },
+        computed: {
+            flyoutSections(): any {
+                return Array.isArray(this.$refs.items) ? this.$refs.items : [this.$refs.items];
+            },
+        },
+        methods: {
+            open(): void {
+                this.$emit('open');
+            },
+            close(): void {
+                this.reset();
+                this.$emit('close');
+            },
+            backToParent(): void {
+                this.$emit('main');
+                this.close();
+            },
+            reset(): void {
+                this.sectionIndex = 0;
+                // @ts-ignore
+                this.flyoutSections.forEach(s => s && s.reset && s.reset());
+            },
+            async selectFirstLink(): Promise<any> {
+                this.reset();
+                await this.$nextTick();
+                // @ts-ignore
+                this.flyoutSections[this.sectionIndex].selectFirstLink();
+            },
+            selectNextLink(e): void {
```

Comment:
Je kunt ook in de aanroep dit doen: @keydown.tab.exact.prevent="selectNextLink" volgens mij, ipv e.preventDefault();